js/app/.eslintrc.yml

```diff
@@ -13,6 +13,7 @@ parserOptions:
     ecmaVersion: 5
     ecmaFeatures:
         jsx: true
+    sourceType: module
```

Comment:
I don't think we can use module here, given ecmaVersion is setting to 5.

Reply:
So should we revert it to use CommonJS instead of ES Modules?

Reply:
I guess so.…（⊙＿⊙；）…
 another way would be adding something like babel, but it needs to be discussed more.

Reply:
I agree - I don't think we should necessarily update to use ES modules here. That seems like a bigger discussion around if we want to use other ES2015+ features or not. I've been away for a bit (so please correce me if I'm wrong!), but I don't think we actually have a real automated build step at the moment.